src/dapp/libexec/dapp/dapp-remappings

```diff
@@ -31,23 +30,23 @@ function findRemappings(prefix) {
       console.error(`${PROGRAM_NAME}: error: try "dapp update" to initialize submodules`)
       process.exit(1)
     }
-    
+
     var hash = run("git", ["-C", src, "rev-parse", "HEAD"])
-  
+
     if (pkg_src[name]) {
       if (hash != pkg_hash[name]) {
-        console.error(`${PROGRAM_NAME}: error: mismatching packages:`)
-        console.error(`${PROGRAM_NAME}: error: (1) ${pkg_src[name]}`)
-        console.error(`${PROGRAM_NAME}: error: (2) ${src}`)
-        process.exit(1)
+        console.error(`${PROGRAM_NAME}: warning: mismatching packages:`)
+        console.error(`${PROGRAM_NAME}: warning: (1) ${pkg_src[name]}`)
+        console.error(`${PROGRAM_NAME}: warning: (2) ${src}`)
+        console.error(`${PROGRAM_NAME}: warning: using ${pkg_src[name]}. You can override this using \`DAPP_REMAPPINGS\``)
```

Comment:
Is this always going to be the across different filesystems / invocations? Bit nervous that we're introducing some non determinism into the build process here....

Reply:
hmm, yeah, not sure

Reply:
added a sort to `ls()`, which I think should be enough to ensure we always have a consistent choice in case of conflicts?

Reply:
nice, sounds good to me!